src/components/Calculator.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Calculator = () => (
+
+  <section className="main-container">
+    <input type="number" placeholder="0" />
+    <div className="clear">A/C</div>
+    <div className="plus-minus">+/-</div>
+    <div className="modulus">%</div>
+    <div className="division orange">÷</div>
+    <div className="seven">7</div>
+    <div className="eight">8</div>
+    <div className="nine">9</div>
+    <div className="multiply orange">x</div>
+    <div className="four">4</div>
+    <div className="five">5</div>
+    <div className="six">6</div>
```

Comment:
- [ ] Great job working on the requirements so far but I think it would be great if you Create at least 1 child component for the calculator as mentioned in the requirements since it is very important to try that to know how the component works in React 😊⭐✅🤩💯

![image](https://github.com/NoelLincoln/Math-Magicians/assets/58553711/81a5e292-bd5a-4548-8831-4ba8b661362e)